src/plugins/template/src/CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ ov_mark_target_as_cc(${TARGET_NAME})
 
 target_include_directories(${TARGET_NAME} PRIVATE
     "${CMAKE_CURRENT_SOURCE_DIR}"
+    "${OpenVINO_SOURCE_DIR}/src/inference/src/dev"
```

Comment:
why is it needed? temp solution until template plugin migrate to new Infer Request?

Reply:
Yes it is temporary solution